plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/lsp/AmazonQLanguageClientImpl.kt

```diff
@@ -308,6 +310,19 @@ class AmazonQLanguageClientImpl(private val project: Project) : AmazonQLanguageC
             ApplicationManager.getApplication()::invokeLater
         )
 
+    override fun sendContextCommands(params: LSPAny): CompletableFuture<Unit> {
+        val showContextCommands = """
+            {
```

Comment:
We have a util for this ChatCommunicationManager.convertNotificationToJsonForChat

Reply:
i dont like that pattern
https://github.com/aws/aws-toolkit-jetbrains/pull/5693